packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {ConsoleLog, Log, Config} from '@bubblewrap/core';
+import {join} from 'path';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+import {enUS as messages} from '../strings';
+
+async function jdkDoctor(config: Config, log: Log): Promise<boolean> {
+  const jdkPath = config.jdkPath;
+  // Checks if the path given is valid.
+  if (!existsSync(jdkPath)) {
+    log.error(messages.jdkPathIsNotCorrect);
+    return false;
+  };
+  try {
+    const file = await fsPromises.readFile(join(jdkPath, 'release'), 'utf-8');
+    if (file.indexOf('JAVA_VERSION="1.8') < 0) { // Checks if the jdk's version is 8 as needed
+      log.error(messages.jdkIsNotSupported);
+      return false;
+    }
+  } catch {
+    log.error(messages.jdkPathIsNotCorrect);
```

Comment:
nit: It might be helpful to append the error message to the error (`e.message`).